examples/gallery/plot/datetime-inputs.py

```diff
@@ -7,14 +7,15 @@
 - :class:`numpy.datetime64`
 - :class:`pandas.DatetimeIndex`
 - :class:`xarray.DataArray`: datetimes included in a *xarray.DataArray*
-- raw datetime strings in `ISO format <https://en.wikipedia.org/wiki/ISO_8601>`__  (e.g. ``"YYYY-MM-DD"``, ``"YYYY-MM-DDTHH"``, and ``"YYYY-MM-DDTHH:MM:SS"``)
+- raw datetime strings in `ISO format <https://en.wikipedia.org/wiki/ISO_8601>``__  (e.g. ``"YYYY-MM-DD"``, ``"YYYY-MM-DDTHH"``, and ``"YYYY-MM-DDTHH:MM:SS"``) 
```

Comment:
```suggestion
- raw datetime strings in `ISO format <https://en.wikipedia.org/wiki/ISO_8601>`__
  (e.g. ``"YYYY-MM-DD"``, ``"YYYY-MM-DDTHH"``, and ``"YYYY-MM-DDTHH:MM:SS"``) 
```

Reply:
Thanks for fixing @seisman. Had some trouble with formatting the URL.